src/components/CommunityEvents/index.tsx

```diff
@@ -0,0 +1,221 @@
+//Libraries
+import React from "react"
+import { useI18next, useTranslation } from "gatsby-plugin-react-i18next"
+import {
+  Box,
+  Center,
+  Divider,
+  Flex,
+  Grid,
+  GridItem,
+  Heading,
+  Icon,
+  Text,
+} from "@chakra-ui/react"
+import { FaDiscord } from "react-icons/fa"
+import { DateTime } from "luxon"
+
+// Components
+import ButtonLink from "../ButtonLink"
+import Link from "../Link"
+import Translation from "../Translation"
+
+// Utils
+import { trackCustomEvent } from "../../utils/matomo"
+
+// Hooks
+import { useCommunityEvents } from "./useCommunityEvents"
+
+const matomoEvent = (buttonType) => {
+  trackCustomEvent({
+    eventCategory: "CommunityEventsWidget",
+    eventAction: "clicked",
+    eventName: buttonType,
+  })
+}
+
+const renderEventDateTime = (
+  date,
+  language,
+  params = {
+    year: "numeric",
+    month: "long",
+    day: "numeric",
+    hour12: false,
+    hour: "numeric",
+    minute: "numeric",
+  }
+) => {
+  return DateTime.fromISO(date).setLocale(language).toLocaleString(params)
+}
+
+const EventLink = (props) => <Link fontWeight="700" {...props} />
+
+const Event = ({ event, language, type }) => {
```

Comment:
Consider creating a separated file for this component, `components/CommunityEvents/Event.tsx`.

```suggestion
interface EventProps {
  event: Event
  language: string
  type: "upcoming" | "past"
}


const Event = ({ event, language, type }: EventProps) => {
```